spec/legacy/unit/journal_spec.rb

```diff
@@ -37,21 +37,19 @@
     end
   end
 
-  it 'should create should send email notification' do
-    ActionMailer::Base.deliveries.clear
+  it 'create should send email notification' do
     issue = WorkPackage.find(:first)
     if issue.journals.empty?
       issue.add_journal(User.current, 'This journal represents the creationa of journal version 1')
       issue.save
     end
-    user = User.find(:first)
-    assert_equal 0, ActionMailer::Base.deliveries.size
```

Comment:
It didn't sound right upon discovery, but with all your opaque black magic going on here I was rather careful with my conclusions ^^